### ISSUE TITLE
Request profile info earlier in registration flow

### ISSUE THREAD

**Issue by pdpinch** [mitodl/mitxonline#1494]
As a marketer, I'd like to get more profile information sooner, when the user registers and before they have enrolled. Knowing users' profiles allows me to target them for enrollment messaging. 

#### Acceptance Criteria:

- [ ] Add a second screen to the registration flow, similar to xPRO. 
- [ ] On the second screen, include the message about collecting data for research purposes
- [ ] Level of Education, the Type checkboxes and the conditional fields should appear on the 2nd screen. All should be optional
- [ ] When a user clicks into a course from their dashboard, display a pop-up if they haven't responded to level of education and type. Now these fields should be required. 
- [ ] Only show the pop-up once to users.

**Comment by jkachel** [comment]
Do the last two checkboxes (When a user clicks into a course... and Only show the pop-up once to users.) count for this one? 

If the user opts not to fill out this data during the signup flow, should we continue to prompt them (only once) when they navigate into a course?

**Comment by pdpinch** [comment]
Yes. The difference is that the 2nd time, some of the fields will be required
